src/authorization.js

```diff
@@ -415,16 +413,16 @@ class Authorization {
     if (this.virtual) {
       return []
     }
-    var statement
-    var fragment = rdf.namedNode('#' + this.hashFragment())
-    var ns = vocab(rdf)
-    var statements = [
+    let statement
+    let fragment = rdf.namedNode('#' + this.hashFragment())
```

Comment:
I captured above in dedicated issue https://github.com/solid/solid-permissions/issues/17 since this PR doesn't really introduce that problem and it can get addressed independently after merning this PR.